package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "my-project",
+  "description": "A Vue.js project",
+  "version": "1.0.0",
+  "author": "jason00111 <emberleyjason@gmail.com>",
+  "private": true,
+  "scripts": {
+    "dev": "cross-env NODE_ENV=development webpack-dev-server --open --inline --hot",
+    "build": "cross-env NODE_ENV=production webpack --progress --hide-modules"
```

Comment:
This micro-lib allows you to provide a script which sets an environment using unix style and have it work on windows too